xml/System.Text.Json/JsonElement+ObjectEnumerator.xml

```diff
@@ -178,8 +180,8 @@ The enumerator will enumerate the properties in the order they are declared, and
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns an enumerator that iterates through a collection.</summary>
+        <returns>An <see cref="T:System.Collections.Generic.IEnumerator" /> object for <see cref="T:System.Text.Json.JsonProperty" /> objects that can be used to iterate through the collection.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.Json.JsonElement.ObjectEnumerator> instance is cast to an [IEnumerator<JsonProperty>](xref:System.Collections.Generic.IEnumerator%601) interface.

         ]]></format>
         </remarks>
```